src/components/admin/UserStatusInput.tsx

```diff
@@ -51,24 +49,27 @@ export const AdminUserStatusInput: React.FC<AdminUserStatusInputProps> = ({
   return (
     <span className="AdminUserStatusInput">
       <span className="AdminUserStatusInput__inputs">
-        <AdminInput
-          {...extraProps}
-          className="AdminUserStatusInput__text"
-          name={inputStatus}
-          register={register}
-        />
-        <input
-          className="AdminUserStatusInput__color"
-          type="hidden"
-          {...register(`userStatuses.${index}.color`)}
-        />
-        <ButtonNG
-          style={pickerStyles}
-          onClick={toggleColorPicker}
-          iconOnly
-          iconName={faEyeDropper}
-        />
-        <ButtonNG onClick={onRemove} iconOnly iconName={faTimes} />
+        <InputGroup withLabel title="Status name">
+          <Input
+            {...extraProps}
+            className="AdminUserStatusInput__text"
+            name={inputStatus}
+            register={register}
+          />
+          <input
+            className="AdminUserStatusInput__color"
+            type="hidden"
+            {...register(`userStatuses.${index}.color`)}
+          />
```

Comment:
This might be the saving status issue, there isn't same hidden input but registered as `userStatuses.${index}.status` and if no `register` is called for such element, the `useForm()` hook just ignores it.